.travis.yml

```diff
@@ -1,21 +1,17 @@
 sudo: false
-dist: trusty
+dist: xenial
 addons:
     apt:
         sources:
-            - deadsnakes
             - ubuntu-toolchain-r-test
         packages:
             - python3.5
             - python3.5-dev
             - python-pip
             - python-virtualenv
-            - gcc-4.9
-            - g++-4.9
```

Comment:
scary compiler upgrade, 
![3573mf](https://user-images.githubusercontent.com/6131403/60837034-7c733680-a1c7-11e9-9133-a7a222a8ccad.jpg)

Reply:
Its only for unit test though. And they pass.